amethyst_rendy/src/camera.rs

```diff
@@ -925,4 +925,55 @@ mod tests {
     fn orthographic_project_cube_off_centered_rotated() {
         unimplemented!()
     }
+
+    #[cfg(not(feature = "nightly"))]
+    #[bench]
+    pub fn transform_global_view_matrix_1000(b: &mut bencher::Bencher) {
+        do_transform_global_view_matrix_1000(b)
+    }
+
+    #[cfg(feature = "nightly")]
+    #[bench]
+    pub fn transform_global_view_matrix_1000(b: &mut test::Bencher) {
+        do_transform_global_view_matrix_1000(b)
+    }
+
+    pub fn do_transform_global_view_matrix_1000<T>(b: T) {
+        let (transform, _, _) = setup();
+
+        b.iter(|| {
+            for _ in 0..1000 {
+                let _: [[f32; 4]; 4] =
+                    convert::<_, Matrix4<f32>>(transform.global_view_matrix()).into();
+            }
+        });
+    }
+
+    #[cfg(not(feature = "nightly"))]
+    #[bench]
+    pub fn manual_inverse_global_matrix_1000(b: &mut bencher::Bencher) {
```

Comment:
You could probably do `type Benecher = bencher::Bencher;` or `test::Bencher` under `cfg`. Those functions differ only by that type. Then the `do_manual_inverse_global_matrix_1000` function can be completely inlined.

Reply:
Also as @torkleyy said above, just put that in `benches/` and problem solved.

Reply:
Done. moved it to benches and swapped to Criterion